src/java.base/share/classes/java/security/KDFParameters.java

```diff
@@ -38,8 +38,12 @@
  * {@link javax.crypto.KDF#getInstance(String, KDFParameters) KDF.getInstance}
  * methods. The {@code getInstance} method returns a {@code KDF}. The
  * {@code KDF} that is returned is initialized with the specified parameters.
+ * <p>
+ * The {@code KDFParameters} used for initialization can be retrieved via
+ * {@link javax.crypto.KDF#getParameters()}.
```

Comment:
This should also mention that the parameters may contain additional values. Suggest changing to:

The {@code KDFParameters} used for initialization are returned by {@link javax.crypto.KDF#getParameters()} and may contain additional default or random parameter values used by the underlying KDF algorithm..